python_modules/libraries/dagster-k8s/dagster_k8s/executor.py

```diff
@@ -35,6 +35,12 @@
         {
             "job_namespace": Field(StringSource, is_required=False),
             "retries": get_retries_config(),
+            "max_concurrency": Field(
+                IntSource,
+                is_required=False,
+                description="Limit on number of concurrent pods launched per run. Note that this is "
```

Comment:
nit: maybe
concurrent pods launched per run -> concurrent pods runnings within the scope of this run